packages/backend-common/config.d.ts

```diff
@@ -69,6 +69,11 @@ export interface Config {
        * Defaults to true if unspecified.
        */
       ensureExists?: boolean;
+      /**
+       * Arbitrary config object to pass to knex when initialising (https://knexjs.org/#Installation-client).
+       * Most notable is the debug and asyncStackTraces booleans
+       */
+      additionalKnexConfig?: object;
```

Comment:
Feels a bit unfortunate to have this be just an `object`, the set of config options from knex is fairly small right? Perhaps worth inlining them here.

We're also thinking that we could probably go with the slightly shorter `knexConfig`, since we're dealing with database connection config in this block, and this would be the knex specific config for that here.